src/Sylius/Bundle/CoreBundle/EventListener/Workflow/OrderCheckout/CompleteCheckoutListener.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\EventListener\Workflow\OrderCheckout;
+
+use Sylius\Component\Core\Model\OrderInterface;
+use Symfony\Component\Workflow\Event\CompletedEvent;
+use Webmozart\Assert\Assert;
+
+final class CompleteCheckoutListener
```

Comment:
If naming scheme follows winzou callbacks, then it should be
```suggestion
final class SaveCheckoutCompletionDateListener
```

Plus `CompleteCheckout` is guaranteed to be misunderstood.